Removed redundant a labelView hidden

wizages opened the pull request:
This could issues where that other tweaks may use this and since you don't check if your tweak is enabled this is hell of bad practice.... Defiantly since you call it after %orig;

wizages committed:
Removed redundant a labelView hidden

This could issues where that other tweaks may use this and since you dont check if your tweak is enabled this is hell of bad practice....

eswick commented:
If you move an app outside of the dock and back to the home screen, the label will remain hidden without this code.

wizages committed:
Update 2

What about this. This pretty much moves the if statement from the bottom to the top and then also checks to make sure the tweak is enabled.

wizages commented:
Try this instead I get what you mean haha.

eswick commented:
Isn't that the same as before, though? What are you trying to accomplish?

wizages commented:
No I moved the if statement into the enable. Currently what your tweak is doing is it is preventing any other tweak from using the label hidden function because you call it after the layout is done being set up. And the worst part is you arent even checking if your tweak is enabled so no one can use this hidden function because you just automatically set it to be not hidden.

eswick commented:
But the bug would still be present when Harbor is enabled, would it not?

wizage committed:
Adding support to label notify

This will add support for label notify. I tested it extensively unlike
the other pull requests.

wizages commented:
I compiled the tweak and made sure it all works. This is a fix that will enable me to fix label notify for my next update!
